### PR TITLE
RK-1303 added whitelist documentation

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

File: docs/breakpoints-tasks.md [hunk 1]
```diff
@@ -26,6 +28,28 @@ For example, adding the following snippet to the Breakpoint Operations section:
...
+The following snippet shows how to get local variables that match the filter.
+```json
+{
+  "name": "filter",
+  "paths": ["store.rookout.frame.locals"],
+  "whitelist": true,
+  "filters": [
+    {
+      "filter_type": "name",
+      "pattern": "secretKey"
```
Comment (Zohar2506) [inline docs/breakpoints-tasks.md]: secretKey is bad example :) use personal name or something

File: docs/breakpoints-tasks.md [hunk 2]
```diff
@@ -26,6 +28,53 @@ For example, adding the following snippet to the Breakpoint Operations section:
...
+	}
+}
+
+```
+
+The `dog` member of any animal instance will be return entirely, while the `numberOfAnimals` will be redacted.
+Changing the `dog` pattern to `name` - will redact the `age` member of dog but not his name.
+
+
+Whitelising by value is not supported.
```
Comment (Zohar2506) [inline docs/breakpoints-tasks.md]: Mention that the whitelisting applies for the *variable name* itself

File: docs/breakpoints-tasks.md [hunk 3]
```diff
@@ -26,6 +28,51 @@ For example, adding the following snippet to the Breakpoint Operations section:
 
 Will replace an instance of "secretKey":"12345" with "secretKey":"[REDACTED]", and an instance of "nameAndPassword":"LordHelmet-12345" with "nameAndPassword":"LordHelmet-****".
 
+### Whitelisting
+
+Whitelisting is the opposite operation of the example above; all the data is redacted except the matched patterns.
+Whitelisting support only variable names.
```
Comment (BenTheKlein) [inline docs/breakpoints-tasks.md]: @Zohar2506

Comment (Zohar2506) [reply docs/breakpoints-tasks.md]: The regex for whitelisting applies for the variable names only. Which means Rookout will look for regex matches to the variable name itself.

image

**Pull request merged**